src/templates/header/index.hbs

```diff
@@ -17,17 +17,18 @@
 
             <div data-nav-container class="c-nav-container">
                 <ul class="c-nav-list">
-                    <li data-auth-wrapper class="c-nav-list-item has-sublist is-hidden" data-test-id="navList" tabindex="0">
+                    <li data-auth-wrapper class="c-nav-list-item has-sublist {{#unless user.isAuthenticated}}is-hidden{{/unless}}" data-test-id="navList" tabindex="0">
```

Comment:
Won't these tags break the handlebars templates for things that don't have tag support for `unless`?

Reply:
I assumed `unless` was a built-in intrinsic capability like `if`. I mean "is not" is a pretty fundamental thing to be able to do...

Reply:
Just checked the docs for handlebars and it seems `unless` [is a built in operator](https://handlebarsjs.com/builtin_helpers.html)!

Learn something new and all that 💡 Ignore me… 😜

Reply:
Ashley who?